src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+
+## Usage
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus to get a solid blue color, you would write this: `color: rgb(var(--color-blue-default));`,
+or like this to get a blue color with 10% opacity: `color: rgba(var(--color-blue-default), 0.1);`
+
+### Available palettes
+In your web-components, you only need to import one of the palettes. Importing a palette makes all included CSS variables available in that web-component. It all depends on how many colors you need for that specific component. If you need more colors than what is offered in the UI palette, then simply remove that and import one of the other palettes that suits.
+
+Each palette also includes Lime Technologies' brand colors. So no matter which one you import, you will always have access to our brand colors and can use them by their specific CSS variable names.
+
+#### **UI palette** (default)
+Includes 10 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-ui-color-palette />
+
+In most cases, importing the UI palette is enough and gives you colors which you normally need in a user interface. This palette includes a few hand picked swatches that add an extra layer of meaning to different elements. Using these, an element in the UI can quickly communicate to the user what it does or indicates.
+
+As you notice, for each hue there is a `-default` and `-light` swatch available in this palette. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
+___
+
+#### **Primary palette**
+Includes 40 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+<limel-example-primary-color-palette />
+
+Sometimes, just a handful of colors is not enough and you need a bit more. For example when you need colors for your icons, or diagrams. This palette is designed to provide a decent number of harmonic colors while easing the choice of colors for you. Used for primary purposes and meaningful communications with colors.
+
+Note that this palette also includes brand colors, and contrast swatches.
+___
+
+#### **Extended palette**
+Includes 100 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-extended-color-palette />
+
+This palette offers a much richer set of colors that can be used for visual communications, iconography, illustrations, and charts.
+
+___
+## Lime Technologies' Brand Colors
```

Comment:
Separate consideration: Should we use just "Lime" instead of "Lime Technologies"?

I noticed on line 91 we just say "Lime's brand assets" and that just sound a lot better than "Lime Technologies' brand assets"…

Reply:
That's a question for the people who set the brand guidelines actually. :)  I don't know. I think to be correct legally, we should say the full name!? Is "Lime" itself a trademark of Lime Technologies AB?

Also for outsiders who have no clue about the people behind lime elements, mentioning the full name may be better.